examples/step_env.py

```diff
@@ -12,17 +13,14 @@
 args = parser.parse_args()
 
 # Construct the environment
-env = gym.make('MountainCar-v0')
+env = GarageEnv(gym.make('MountainCar-v0'), max_episode_length=args.n_steps)
```

Comment:
hmm probably don't do that? because you might step past the terminal state

Reply:
`GymEnv` should produce `es.step_type == StepType.TERMINAL` if `max_episode_length` is reached. `es.last` should capture that.